packages/react-vapor/src/components/table-hoc/TableRowConnected.tsx

```diff
@@ -190,9 +190,9 @@ class TableRowConnected extends React.PureComponent<
     };
 
     private handleDoubleClick = () => {
-        _(this.props.actions)
-            .filter((action: IActionOptions) => action.callOnDoubleClick)
-            .forEach((action: IActionOptions) => action.trigger());
+        _(this.props.actions).each((action: IActionOptions) =>
+            action.callOnDoubleClick && action.link ? window.location.replace(action.link) : action.trigger()

```

Comment:
Since `location.replace` [replaces the current history](https://developer.mozilla.org/en-US/docs/Web/API/Location/replace), I think that you won't be able to navigate back.

I think it would be more useful with `window.location.href = action.link` ?

Reply:
`window.location.href = action.link ` brings me to `localhost/action.link` I don't know why.

And I would seem that `location.replace` is superior exactly because the page is not kept in the history. I think I was able to go back though... but I'll test again to be sure.

good point.